### ISSUE TITLE
[BUG] `--boot-to-virtual-media` reports error on PowerEdge R6525

### ISSUE THREAD

**Issue by grafuls** [redhat-performance/badfish#354]
**Your System Details**

* Python Version: Python 3.10.9
* Operating System: Fedora release 36 (Thirty Six)
* Target System Type: (_e.g. Dell, SuperMicro_): PowerEdge R6525

**Describe the bug**
When trying to boot to a mounted virtual media BF returns an error message where it complains about not being able to create BIOS config job.

**To Reproduce / What were you Doing?**
1. Mount a virtual media via `--mount-virtual-media`
2. Subsequently run `--boot-to-virtual-media`
3. Check output

**Expected Behavior**
No error message is returned and the system boots successfully to the Virtual Media

**Logs / Screenshots**

```
✘ badfish -H {HOST} -u root -p {PASS} --boot-to-virtual-media
- WARNING  - Job queue already cleared for iDRAC {HOST}, DELETE command will not execute.
- INFO     - Command passed to set BIOS attribute pending values.
- ERROR    - POST command failed to create BIOS config job, status code is 400.
- ERROR    - Pending configuration values are already committed, unable to perform another set operation.
```

**Additional Details**
Looks like we don't really need to create a BIOS configuration job and instead just attempt the reboot after sending the one-time-boot instructions as seen on this part of the [code](https://github.com/redhat-performance/badfish/blob/5d3cc181e61e3f3fede15ab9e4fc4420ffe7a8b3/src/badfish/main.py#L1071). Maybe worth to check if this case is only happening on this specific model.

**Comment by jmencak** [comment]
Additional information, I'm getting this even on Dell 740xd.

**Comment by k3nd1k** [comment]
This should be fixed with the above mentioned PR #373.

> The reported problem/bug was found to be present only on Dell systems. It appeared when forcing a one time boot to a specific device and virtual media.
> This functionality of commands using this approach was still right/valid (after rebooting the server it booted to the specified device), only the CLI output/log was displaying errors.
> The cause of this was found to be a call of create_bios_config_job() right after send_one_time_boot() in the boot_to() function. The errors were happening because iDRAC already created the needed config job since the patch request was called with specified apply time to on reset (payload_patch = {"@Redfish.SettingsApplyTime": {"ApplyTime": "OnReset"}}).
> Thus that call was removed since it wasn't needed and the functionality was tested to still be valid.
